crates/rattler_conda_types/src/repo_data/mod.rs

```diff
@@ -357,6 +357,7 @@ impl PackageRecord {
         topological_sort::sort_topologically(records)
     }
 
+    #[allow(clippy::result_large_err)]
```

Comment:
I think you can Box the error type that is large to make this go away (cc @baszalmstra to hear his opinion).

Reply:
Yeah boxing is most often the right approach. I just dont understand how this changed since previous versions?